cvat-ui/src/components/task-page/details.tsx

```diff
@@ -377,7 +385,14 @@ export default class DetailsComponent extends React.PureComponent<Props, State>
                 <Col>
                     <Text strong className='cvat-text-color'>Issue Tracker</Text>
                     <br />
-                    <Text editable={{ onChange: onChangeValue }}>Not specified</Text>
+                    <Text
+                        editable={{
+                            onStart,
+                            onChange: onChangeValue,
+                        }}
+                    >
+                        {bugTrackerEditing ? '' : 'Not specified'}
+                    </Text>
```

Comment:
There different behavior when we press by mouse out of input field and when we press ``Escape`` key.